### ISSUE TITLE
Zotero not showing in the applications list

### ISSUE THREAD

**Issue by meminens** [retorquere/zotero-deb#92]
I installed Zotero via

```
wget -qO- https://raw.githubusercontent.com/retorquere/zotero-deb/master/install.sh | sudo bash
sudo apt update
sudo apt install zotero
```

However, the app is not showing in the applications list (launcher). I am using X11, GNOME 43 on Debian 12.

I can launch the app by typing `zotero` in the Terminal. But I was hoping to be able to launch it from the applications list.

I confirm `zotero.desktop` exists in `/usr/share/applications/` after the install.

**Comment by retorquere** [comment]
Is there a file `/usr/share/applications/zotero.desktop`? Are there other `.desktop` files in that directory?

**Comment by misaligar** [comment]
> Are there other `.desktop` files in that directory?

Yes. Lots of them. Here is a portion of them:

![image](https://github.com/retorquere/zotero-deb/assets/42714627/c587721f-5bef-4e01-ac45-993094b181af)

**Comment by retorquere** [comment]
Does `xdg-desktop-menu forceupdate` make it show up?

**Comment by misaligar** [comment]
> xdg-desktop-menu forceupdat

Nope. Here is another screenshot to illustrate the problem.

![image](https://github.com/retorquere/zotero-deb/assets/42714627/8eb0a3d1-08c3-4c95-99fb-5e355b0ee132)

I expect Zotero to show next to Zoom. When I type Zotero fully, it doesn't show any apps. Only the files/folders named Zotero.

**Comment by misaligar** [comment]
Don't if this helps but Zotero can't be pinned to Dash either.

https://github.com/retorquere/zotero-deb/assets/42714627/1f586cea-fd5a-4d2d-8742-ac1c7106a874

**Comment by misaligar** [comment]
here is an update. I was able to fix the issue by renaming `zotero.desktop` to `zotero_2.desktop`. I don't know how this fixes the issue but everything seems to be in order. It doesn't matter what the file name is. It just has to be something different from `zotero.desktop`.

**Comment by retorquere** [comment]
Is there another zotero.desktop on your system? In your home directory perhaps?

**Comment by misaligar** [comment]
> Is there another zotero.desktop on your system? In your home directory perhaps?

Nope. I just ran `sudo find / -type f -name zotero.desktop` to confirm. Only the one in `/usr/share/applications`.

**Comment by retorquere** [comment]
Then I really don't know what else it could be. It seems improbable that this particular name would be forbidden on Debian.

**Comment by misaligar** [comment]
Thank you so much for your time. I really wish I could figure out why this happens but sounds like we have exhausted troubleshooting options here. I am closing this issue.

**Comment by Kingofhollows099** [comment]
I have the same problem. Followed the troubleshooting events mentioned earlier in the issue. 

- I had previously attempted installing with tarball, but switched to this. I did remove the Zotero desktop file at ~/.local/share/.
- Running `sudo find / -type f -name zotero.desktop` returns: 

> /usr/share/applications/zotero.desktop
> /usr/lib/zotero/zotero.desktop
> /mnt/Silo/.Trash-1000/expunged/3316321444/zotero.desktop
> find: ‘/run/user/1000/gvfs’: Permission denied
> find: ‘/run/user/1000/doc’: Permission denied

**Comment by Kingofhollows099** [comment]
I was about to try that, and I realized that for some reason, Zotero is showing up now. I can't tell you why, but it fixed on its own.

My laptop has died twice since then, so it may have been me restarting. I thought I restarted to try and fix it before, but I might not have.
